.github/workflows/release.yml

```diff
@@ -0,0 +1,226 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v[0-9]+.*
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Release Version
+        default: v2.1.0
```

Comment:
Can this be deduced from the tag?

Reply:
That is just a place holder that gets pre-filled for you when manually triggering the workflow. look at the screenshot on first post, it could be replaced by vx.x.x